### ISSUE TITLE
openvr_api.dll and openvr64_api.dll to allow 32bit and 64bit version to co-exist

### ISSUE THREAD

**Issue by erwincoumans** [ValveSoftware/openvr#412]
Right now, both 32bit and 64bit version of the Windows dll are called openvr_api.dll.
This makes it difficult to use both at the same time. Could you consider renaming the 64bit version, like openvr64_api.dll?

**Comment by erwincoumans** [comment]
Yes, renaming the 32bit version would do too. For now, I hacked the binary openvr_api.lib, to search for a different name. I hope someone has time to look into it.

**Comment by jrbudda** [comment]
and maybe version information too?

**Comment by cmbruns-hhmi** [comment]
This would save me the minor inconvenience of renaming these libraries every time there is a new release, when I update the python bindings
https://github.com/cmbruns/pyopenvr/tree/master/src/openvr
